docs/products/web-components/library/sesamy-address.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 11
+---
+
+# sesamy-address
+
+The address component displays and allows updating the address of the current vendor user. It could for instance be used on a user profile website
+
+## Usage
+
+### Basic usage
+
+```html
+<html>
+  <head></head>
+  <body>
+    <sesamy-address></sesamy-address>
+
+    <script
+      defer
+      src="https://assets.sesamy.com/scripts/web-components/sesamy-address.min.js"
```

Comment:
I included it in the webpack common file to build the standalone component file (https://github.com/sesamyab/web-components/blob/master/webpack/webpack.common.js#L15) but maybe it could cause some confusion because we're using the bundle in some places on the doc, right?

Reply:
We had some issues with publishers importing multiple components separately and since we're not guarding for multiple duplicated registrations of components it gets a bit tricky (one component could include another one..)